Adafruit_PWMServoDriver.cpp

```diff
@@ -64,7 +63,10 @@ Adafruit_PWMServoDriver::Adafruit_PWMServoDriver(const uint8_t addr,
  *          Sets External Clock (Optional)
  */
 void Adafruit_PWMServoDriver::begin(uint8_t prescale) {
```

Comment:
lets change this to return bool? that way we can test finding/success

Reply:
sure. was thinking same.